apps/builder/app/canvas/shared/styles.ts

```diff
@@ -66,15 +66,15 @@ const helperStyles = [
     box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.7);
   }`,
   // Has no width, will collapse
-  `[${idAttribute}]:where(:not(body)[${collapsedAttribute}="w"]) {
+  `[${idAttribute}]:where(:not(body):not([data-lexical-editor])[${collapsedAttribute}="w"]) {
```

Comment:
this selector slowly get to wtf levels, maybe time to think of a simplification

Reply:
please add comment what this is fixign, there is no way we will understand later on

Reply:
Done.

Reply:
comment says what it does, but not why

Reply:
Fixed.